server/helpers/findLendingHistory.js

```diff
@@ -0,0 +1,29 @@
+import jwt from 'jsonwebtoken';
+import models from '../database/models';
+import responseMessage from './responseMessage';
+
+const { LendingHistory } = models;
+
+const findLendingHistory = async (id, bookId, response) => {
+  try {
+    let bookDuration = '';
+    const lentBook = await LendingHistory.findOne({ where: { userId: id, bookId, type: 'borrowed' }, order: [['createdAt', 'DESC']], limit: 1 });
```

Comment:
I can see there is a column called type in the LendingHistory table. What other value does it hold apart from 'borrowed' and where was it used in this PR

Reply:
Books can be borrowed or bought, but this route is for borrowed books only